src/OrleansServiceBus/Providers/Streams/EventHub/EventHubDataAdapter.cs

```diff
@@ -44,6 +44,30 @@ public struct CachedEventHubMessage
     public class EventHubMessage
     {
         /// <summary>
+        /// Contructor
+        /// </summary>
+        /// <param name="streamIdentity"></param>
+        /// <param name="partitionKey"></param>
```

Comment:
Not sure how important, but I've been suggested to use the [constructor doc string](http://stylecop.soyuz5.com/SA1642.html) here: https://github.com/dotnet/orleans/pull/2223#pullrequestreview-2441730
Also, please add description for the properties (or remove them entirely, but looks like we could use a description here).